test/user.test.js

```diff
@@ -0,0 +1,57 @@
+const request = require('supertest');
+
+const app = require('../src/backend/web/app');
+
+jest.mock('../src/backend/utils/elastic');
+
+// Mock the internal authentication strategy
+jest.mock('../src/backend/web/authentication');
+// Use our authentication test helper
+const { login, logout } = require('./lib/authentication');
+
+describe.skip('test GET /user/feeds endpoint', () => {
+  let user;
+
+  beforeAll(() => {
+    user = login('Johannes Kepler', 'user1@example.com');
+  });
+
+  beforeEach(() => {
+    logout();
+    login('Johannes Kepler', 'user1@example.com');
```

Comment:
Change this to be:

```js
user = login('Johannes Kepler', 'user1@example.com');
```

Reply:
Fixed in https://github.com/Seneca-CDOT/telescope/pull/958/commits/5cee6b4c37a89f88fa372ea01b50ff1e5cf45b72